### ISSUE TITLE
{resolution} results in the output of 'premium' when TubeSync downloads a Premium source.

### ISSUE THREAD

**Issue by AshleyAshes** [meeb/tubesync#499]
I didn't even realize that TubeSync could download the 'Premium' flagged encodes, but nifty.  That said it's not giving me a resolution from the {resolution}.

I'm currently using this formatting:

{source_full} {yyyy_mm_dd} {title_full} [{resolution} WEB-DL].{ext}

And now have some files coming in like this:

Nostalgia Nerd 2021-11-17 This Mysterious Computer Could Prove Time Travel Exists  Nostalgia Nerd [premium WEB-DL].mkv

'Premium' doesn't really tell me what resolution the file is, since, AFAIK, Premium could be any number of resolutions, depending on what the highest uploaded resolution is.

**Comment by meeb** [comment]
Thanks, there's obviously been some changes on the YouTube side that I need to update in tubesync. tubesync itself doesn't "support" premium streams, it's really just a web UI wrapper around yt-dlp which does all of the heavy lifting so it'll support whatever yt-dlp upstream supports. It has some logic in it for the file naming though which is probably where this issue is hiding. I don't have a YouTube account myself so issues with premium flagged formats generally require people like you to report them here as I can't stumble over and see them myself.

I'll look into this before the next release.

**Comment by AshleyAshes** [comment]
Yeah, so about that 'YouTube Account' thing.  I've done nothing to enter my YT account info into TubeSync and I don't pay for YT premium anyway.  I don't know how it downloaded something 'Premium' if it really did,

**Comment by meeb** [comment]
Ah fair enough. It's likely that YouTube are occasionally presenting "premium" streams for non-premium and not authenticated users then, or at least offering them as formats which you can't access. Either way it's a change at YouTube which will need investigation and updating in tubesync to account for it. Cheers for the details.

**Comment by gilangjavier** [comment]
I can help fix this. I’ll investigate the naming logic for {resolution} (and the unexpected fps suffix) and submit a PR once it’s done.
